java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -23,7 +23,8 @@
 import java.time.ZoneOffset;
 import java.time.chrono.ChronoLocalDate;
 import java.time.format.DateTimeFormatter;
-import org.apache.commons.lang.ArrayUtils;
+
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
Just a question. Is it required? Or, can we do this first as a small PR?

Reply:
Cleaned up